contributing/process-outline.md

```diff
@@ -34,26 +33,35 @@ you can choose one that you're interested in working on.
   5. Telemetry
   6. Release post/Demo (except for small doc changes/fixes). 
 - Once the scope is agreed, you can start working on the pull request. 
-- Each issue/PR should have a main reviewer, if you didn't see anyone aligned, 
-  please ask in the issue and tag us.
+- Unless otherwise noticed, the person who opened the issue should give the thumbs up for the acceptance criteria (the only exception is when someone outside of our team opened an issue in an open-source project, in such case, someone from the team should approve the acceptance criteria)
 - Once the criteria is set, update the pull request text to have that criteria as a check-list
   both you and the reviewer can reference.
 
-## 3. Submit a Pull Request:
-- Once you've completed the work, submit a pull request to the repository. 
+## 3. Submit a Draft Pull Request
+
+```{note}
+For more information, see [](submitting-pr.md).
+```
+
+- Once you've completed the work, submit a draft pull request to the repository. 
```

Comment:
We can mention here that `draft` is when it's still work in progress. Need to mark `ready for review` when work is completed.

Reply:
fixed

Reply:
Few typos here:
`opem`, `clicck`